src/components/Product/Product.js

```diff
@@ -16,6 +16,7 @@ const Product = product => {
         <span>{product.package}</span>
       </div>
     </div>
+    <button type="submit">Add</button>
```

Comment:
Hello... I liked your idea about passing props to the button so it can be an Add/Remove button depending on whether an item is already in the wishlist. But maybe that's more of a nice to-do?